test/Makefile

```diff
@@ -18,8 +18,7 @@ FIREBASE_DIR=..
 GTEST_DIR=googletest/googletest
 ARDUINOJSON_DIR=../src/third-party/arduino-json-5.3
 
-FIREBASE_SRCS=${FIREBASE_DIR}/src/FirebaseObject.cpp\
-	${FIREBASE_DIR}/src/FirebaseObject.h
+FIREBASE_SRCS=${FIREBASE_DIR}/src/FirebaseObject.cpp
```

Comment:
If you have the .h here when you run "make clean" it deletes it. We also don't need it.